x-pack/plugins/ml/public/application/services/ml_api_service/data_frame_analytics.ts

```diff
@@ -72,6 +72,14 @@ export const dataFrameAnalytics = {
       body,
     });
   },
+  updateDataFrameAnalytics(analyticsId: string, updateConfig: any) {
```

Comment:
Minor suggestion to have UpdateConfig as a type/interface

Reply:
Good call - added in d178c4ca4b4e18e59f34f56e3ba4ad1b6986a800